lib/Conversion/TorchToLinalg/Pooling.cpp

```diff
@@ -1614,6 +1621,459 @@ class ConvertAtenAdaptivePoolOp : public OpConversionPattern<OpTy> {
     return success();
   }
 };
+
+struct ConvertRoiAlignOp final
+    : OpConversionPattern<Torch::TorchvisionRoiAlignOp> {
+  using OpConversionPattern::OpConversionPattern;
+
+  static SmallVector<Value>
+  coordinateTransform(OpBuilder &b, Torch::TorchvisionRoiAlignOp op,
+                      Location loc, SmallVector<Value> outputSizes, Value input,
+                      SmallVector<Value> inputSizes,
+                      SmallVector<Value> scaleValues, std::string coordStr,
+                      bool alignCornersBool, SmallVector<Value> indices,
```

Comment:
Passing vectors by value will result in copies. Instead, you can pass them as `ArrayRef`